session/src/test/java/uk/gov/di/ipv/cri/common/api/handler/SessionHandlerTest.java

```diff
@@ -40,16 +40,14 @@
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyString;
 import static org.mockito.ArgumentMatchers.eq;
-import static org.mockito.Mockito.never;
-import static org.mockito.Mockito.times;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
Please could we use explicit imports rather than wild-cards

Reply:
Done